### ISSUE TITLE
API doc problems of sigmoid_cross_entropy_with_logits

### ISSUE THREAD

**Issue by ranqiu92** [PaddlePaddle/Paddle#9187]
#### 单词拼写、遗漏和语法等问题未列出，修改时请注意
*请参考[API注释标准](https://github.com/PaddlePaddle/Paddle/blob/develop/doc/fluid/dev/api_doc_std_cn.md)！*
------
源文件：https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/layers/ops.py

- sigmoid_cross_entropy_with_logits
http://www.paddlepaddle.org/docs/develop/api/en/fluid/layers.html#sigmoid-cross-entropy-with-logits
  - 缺 Examples
  - 格式问题（数据类型介绍， returns 等）
  - 介绍欠佳，请重新组织语言
  - 显示问题
![image](https://user-images.githubusercontent.com/31875089/37584008-b754802a-2b8e-11e8-8153-5c38ab7f8874.png)
